### DISCUSSION TITLE
lost github streak

### DISCUSSION THREAD

**Discussion by Striver-3110** [community/community#112530] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

i lost my github streak by making some of my repos private.
help me out to recover the same
i have made changes to its visibility to public but still it isnt recovered.

**Comment by ngljcb** [comment, marked as answer]
you can try to make a commit so that it triggers the streak counter again.

**Reply by Striver-3110** [reply]
it worked for me!
thank you!

**Reply by ngljcb** [reply]
no problem! happy to help!

**Reply by pranjalisr** [reply]
Didn't worked for me 🥲
